src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java

```diff
@@ -0,0 +1,30 @@
+package com.example.naitei19javaecommerce.controller.user;
+
+import com.example.naitei19javaecommerce.model.CartItem;
+import com.example.naitei19javaecommerce.service.CartItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Controller
+public class CartController {
+    @Autowired
+    private CartItemService cartItemService;
```

Comment:
```suggestion
    private CartService cartService;
```
ủa cart chứ sao lại cart item dc bé ơi